### ISSUE TITLE
Please add more info on annotations

### ISSUE THREAD

**Issue by RudolfWeeber** [MicrosoftDocs/visualstudio-docs#3969]
It is not clear to me, how to access the content of "squiggles" mentioned in the annotations section.
Pressing CTRL-K CTRL-M or CTRL-K CTRLG-G does nothing in a C++-file with missing semicolons. Tried with NVDA 2019.2. Please extend the information on how the annotations feature should be used.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 378274af-75f0-8696-cc1a-5067834c6dd8
* Version Independent ID: 560d03f2-e5e8-44a2-8794-7e7bfe15f322
* Content: [Accessibility tips and tricks for Visual Studio - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/ide/reference/accessibility-tips-and-tricks?view=vs-2019#feedback)
* Content Source: [docs/ide/reference/accessibility-tips-and-tricks.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/ide/reference/accessibility-tips-and-tricks.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-general**
* GitHub Login: @TerryGLee
* Microsoft Alias: **tglee**

**Comment by WilliamAntonRohm** [comment]
@RudolfWeeber -- Rudolf, thanks for your question. Here are a few options where you might consider asking it: 
-	[https://social.msdn.microsoft.com/Forums/vstudio](https://social.msdn.microsoft.com/Forums/vstudio)
-	[https://www.stackoverflow.com](https://www.stackoverflow.com)
-	[https://www.reddit.com/r/VisualStudio/](https://www.reddit.com/r/VisualStudio)

As well, you might also consider using the [Report a Problem](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio) tool to report the issue.

@TerryGLee -- Terry, please look into adding this information.

**Comment by DanteGagne** [comment]
Hey there @RudolfWeeber , 

Thanks for the feedback. We've got a known issue currently with annotations not being read correctly by either JAWS or NVDA... although they seem to be for different reasons. We're investigating this. However, I do agree that we should update the documentation. I'll make sure that once we've got the feature working again, we get the documentation brought up to date.

**Comment by TerryGLee** [comment]
@DanteGagne Any updates?

**Comment by TerryGLee** [comment]
@RudolfWeeber Thanks for taking the time to report this issue to us. I checked in with @DanteGagne and he reports that the problem you described with annotations not being read correctly by either JAWS or NVDA has been fixed; he tested it with Narrator he reports that it works as expected. We'll close this issue now; but, if you think there are further content updates we should consider for the [Accessibility Tips & Tricks](https://docs.microsoft.com/visualstudio/ide/reference/accessibility-tips-and-tricks?view=vs-2022) page, please open a new issue and let us know how we can help.
